gtk/src/subclass/application.rs

```diff
@@ -82,13 +82,12 @@ unsafe impl<T: GtkApplicationImpl> IsSubclassable<T> for Application {
         }
 
         <gio::Application as IsSubclassable<T>>::override_vfuncs(class);
-        unsafe {
-            let klass = &mut *(class.as_mut() as *mut gtk_sys::GtkApplicationClass);
-            klass.window_added = Some(application_window_added::<T>);
-            klass.window_removed = Some(application_window_removed::<T>);
-            // Chain our startup handler in here
-            let klass = &mut class.as_mut().parent_class;
-            klass.startup = Some(application_startup::<T>);
-        }
+
+        let klass = class.as_mut();
+        klass.window_added = Some(application_window_added::<T>);
+        klass.window_removed = Some(application_window_removed::<T>);
+        // Chain our startup handler in here
+        let klass = &mut class.as_mut().parent_class;
```

Comment:
This could also become `let klass: &mut gio_sys::GApplicationClass = class.as_mut()` but that seems more complicated :)